### PR TITLE
Create Feedback Models and APIs

### PR DESCRIPTION
**Pull request by gsvr**

### CODE REVIEW

**Comment by gsvr** [comment]
@imsickofmaps Please review.

File: trainings/tests.py [hunk 1]
```diff
@@ -190,3 +190,106 @@ def test_api_create_attendee(self):
...
+            response.data["event"],
+            "http://testserver/api/v1/events/%s/" % event.id)
+        self.assertEqual(
+            response.data["participant"],
+            "http://testserver/api/v1/participants/%s/" % participant.id)
+
+    def test_api_create_feedback(self):
+        # Setup
+        # create a trainer
+        trainer_data = {
```
Comment (imsickofmaps) [inline trainings/tests.py]: You wanna mop?

**Comment by imsickofmaps** [comment]
DRY up the test helpers then :+1:

**Comment by gsvr** [comment]
@imsickofmaps I may have mopped too aggressively, but hey :smile:

**Pull request merged**